src/components/CourseChecklist/CourseChecklist.test.jsx

```diff
@@ -0,0 +1,175 @@
+import { Icon } from '@edx/paragon';
+import React from 'react';
+
+import CourseChecklist from '.';
+import getFilteredChecklist from '../../utils/CourseChecklist/getFilteredChecklist';
+import getValidatedValue from '../../utils/CourseChecklist/getValidatedValue';
+import { shallowWithIntl } from '../../utils/i18n/enzymeHelper';
+
+// generating test checklist to avoid relying on actual data
```

Comment:
I'm unclear on why moving the logic to another file would prevent the mocking on lines 28-29? It would change the path sent to those mock calls, but does it also affect it's usability?